### PR TITLE
Updates to flex-config README and CONTRIBUTING

### PR DESCRIPTION
**Pull request by joto**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -6,20 +6,20 @@ We operate the "Fork & Pull" model explained at
...
 
-To understand the osm2pgsql code, it helps to know some history on it. Osm2pgsql
-was written in C in 2007 as a port of an older Python utility. In 2014 it was
-ported to C++ by MapQuest and the last C version was released as 0.86.0. In it's
-time, it has had varying contribution activity, including times with no
-maintainer or active developers.
+To understand the osm2pgsql code, it helps to know some history on it.
+Osm2pgsql was written in C in 2007 as a port of an older Python utility. In
+2014 it was ported to C++ by MapQuest and the last C version was released as
+0.86.0. In it's time, it has had varying contribution activity, including times
```
Comment (lonvia) [inline CONTRIBUTING.md:18]: it's -> its

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -6,20 +6,20 @@ We operate the "Fork & Pull" model explained at
...
+with no maintainer or active developers.
 
-Parts of the codebase still clearly show their C origin and could use rewriting
-in modern C++, making use of data structures in the standard library.
+There are still a few parts that show their C origin, but most of the code is
+now modern C++. We are currently targeting C++17.
```
Comment (lonvia) [inline CONTRIBUTING.md:22]: Far too modest. "Very few parts of the code now show their C origin, most has been transformed to modern C++."

File: CONTRIBUTING.md (hunk 1 above)
Comment (joto) [reply CONTRIBUTING.md]: Fixed.

File: CONTRIBUTING.md (hunk 2 above)
Comment (joto) [reply CONTRIBUTING.md]: Fixed

**Pull request merged**